src/js/controllers/imageCtrl.js

```diff
@@ -0,0 +1,27 @@
+(function (angular) {
+    angular
+        .module('flickrPOC')
+        .controller('imagesCtrl', imagesCtrl);
+
+    imagesCtrl.$inject = ['images', '$log'];
+
+    function imagesCtrl(images, $log) {
+        var vm = this;// jshint ignore:line
+
+        function activate(){
+
+            images.loadPics()
+                .then(function (res) {
+                    vm.images = res;
+                    return vm.images;
+                })
+                .catch(function(err){
+                    $log.error('Error loading images ', err.status, err.data);
+                });
+        }
+
+        activate();
+
+    }
+
+})(angular);
```

Comment:
Good looking controller. Very little code, not complex logic, and testable. Great job.